manifests/klusterlet/managed/klusterlet-registration-clusterrole-addon-management.yaml

```diff
@@ -1,12 +1,13 @@
-# Adddition ClusterRole permission for registartion agent 
-# Registartion agent needs these permission to manage addons
+# Addition ClusterRole permission for registration agent 
```

Comment:
wrong place?

Reply:
Because the namespaces of the subjects service account are different, I think we need to keep this file too.

Reply:
let us be more specific. 

Registration agent needs these permission to manage addons whose agents runs in managed cluster while registration agent itself runs in the management cluster.